src/lucky_record/validations.cr

```diff
@@ -26,4 +26,26 @@ module LuckyRecord::Validations
       field.add_error message
     end
   end
+
+  private def validate_size_of(field, *, is exact_size, message = "is invalid")
```

Comment:
Can the `*` be removed? Maybe I'm missing something, but I'm not sure I understand it

Reply:
This was necessary to allow the subsequent `def validate_size_of` to overload. The `*` splat means all subsequent params need to be passed as named params.

To be explicit, I could add the splat to the second definition as well?

Reply:
Oh cool! I didn't realize you could do that.